### ISSUE TITLE
textureQueryLOD was not supported at #extension ARB_texture_query_lod.

### ISSUE THREAD

**Issue by ZhiqianXia** [KhronosGroup/glslang#2713]
textureQuery**LOD**  function was supported at ARB_texture_query_lod , and textureQuery**Lod**  is was supported  at the core specification. But the glslang didn't Support the  textureQuery**LOD**  functions. By the way, The ARB_texture_query_lod extension had merged into glslang. https://github.com/KhronosGroup/glslang/pull/2194

https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_texture_query_lod.txt says:

The "Lod" spelling is the correct spelling for the core specification and the preferred spelling for use. However, use of
"LOD" also exists, as the extension predated the core specification, so this extension won't remove use of "LOD".

**shader**

```
#version 150

#ifdef GL_ARB_texture_query_lod
#extension GL_ARB_texture_query_lod : enable
#endif
#ifdef GL_ARB_gpu_shader5
#extension GL_ARB_gpu_shader5 : enable
#endif

#ifdef GL_ES
precision highp float;
#endif

in vec2 vUV; // vert->frag
out vec4 color; // frag->fb
#define UV vUV

#define bias 1.5
#define TEX 128.0
#define offset ivec2(1,1)
uniform highp sampler2DShadow sampler;
uniform int funct;

void main (void)
{
    switch (funct)
    {
    case 0:
        ivec2 iv2 = textureSize(sampler, 0);
#ifdef GL_ARB_texture_query_lod
        vec2 fv2 = textureQueryLOD(sampler, vec2(0.0, 0.0));
#endif
		color = vec4(iv2,fv2);
        break;
    default:
        color = vec4(1.0, 1.0, 1.0, 1.0);
        break;
    }
}
```

**Comment by ZhiqianXia** [comment]
@greg-lunarg kindly review the PR.

**Comment by greg-lunarg** [comment]
Fixed by #2723
